readthedocs/restapi/views/model_views.py

```diff
@@ -33,6 +34,28 @@
 log = logging.getLogger(__name__)
 
 
+class PlainTextBuildRenderer(BaseRenderer):
+
+    """
+    Custom renderer for text/plain format.
+
+    charset is 'utf-8' by default.
+    """
+
+    media_type = 'text/plain'
+    format = 'txt'
+
+    def render(self, data, accepted_media_type=None, renderer_context=None):
+        renderer_context = renderer_context or {}
+        response = renderer_context.get('response')
+        if not response or response.exception:
```

Comment:
If there is no response probably is something wrong, anyway, this is just an extra protection since `renderer_context` "always" has this.